package.json

```diff
@@ -21,13 +22,17 @@
   "homepage": "https://github.com/JSerZANP/relay-typescript-tutorial#readme",
   "dependencies": {
     "babel-plugin-relay": "^14.1.0",
-    "graphql": "^16.6.0",
+    "cors": "^2.8.5",
+    "express": "^4.18.1",
+    "express-graphql": "^0.12.0",
+    "graphql": "^15.8.0",
```

Comment:
express-graphql doesn't support the latest GraphQL, so need to downgrade it to 15
it should still work for tutorial purpose